cibuildwheel/linux.py

```diff
@@ -75,7 +76,7 @@ def get_python_configurations(build_selector):
     return [c for c in python_configurations if matches_platform(c.identifier) and build_selector(c.identifier)]
 
 
-def build(project_dir, output_dir, test_command, before_test, test_requires, test_extras, before_build, build_verbosity, build_selector, repair_command, environment, manylinux_images, dependency_constraints):
+def build(opt: BuildOptions):
```

Comment:
I'm not a fan of abbreviations like this, I'd prefer `options` I think

Reply:
If that's too long to type every time, then we could say `project_dir=options.project_dir`.
Then again, what's the point of passing 1 argument, if you immediately expand it again :-/